hw/ip/otbn/dv/otbnsim/sim/state.py

```diff
@@ -35,8 +35,8 @@ def __init__(self) -> None:
         self.dmem = Dmem()
 
         # Stalling support: Instructions can indicate they should stall by
-        # returning false from OTBNInsn.pre_execute. For non instruction related
-        # stalls setting self.non_insn_stall will produce a stall.
+        # returning a non-None value from OTBNInsn.execute. For non instruction
```

Comment:
how about something like "making execute() a generator"? Generators aren't really "returned" in a classical sense (e.g. using `return`), even though that's effectively what happens once `yield` shows up in a function.

Reply:
Oh, that comment is just stale. Thanks for spotting it.